solidity/DieselPrice.sol

```diff
@@ -4,31 +4,30 @@
     This contract keeps in storage a reference
     to the Diesel Price in USD
 */
-pragma solidity ^0.4.0;
-
-import "github.com/oraclize/ethereum-api/oraclizeAPI.sol";
-
+pragma solidity ^0.4.25;
+import "https://raw.githubusercontent.com/oraclize/ethereum-api/master/oraclizeAPI_0.4.25.sol";
 
 contract DieselPrice is usingOraclize {
 
-    uint public dieselPriceUSD;
+    uint256 public dieselPriceUSD;
 
-    event NewOraclizeQuery(string description);
-    event NewDieselPrice(string price);
+    event NewOraclizeQuery(string _description);
```

Comment:
Trying to reach consensus on whether or not to have the trailing underscore in the event params:

__For__: Follows the python convention of internal/external variable explicity. 

__Against__: Looks ugly. (Literally my only argument!)

Thoughts?

Reply:
This is also something that I've been trying to figure out as well. The Solidity Style Guide (https://solidity.readthedocs.io/en/v0.4.25/style-guide.html) makes no mention of leading underscores for function parameters, and the OpenZeppelin contracts also do not use this standard. However, it's something I've seen a few times, and although it is nice for reading (for me personally), I've also noticed that it conflicts very quickly if leading underscores are also used for private/internal variables (i.e. reserving the non-underscore version for get functions that are semantically similar to public variables).

I thought it might be worth suggesting, and would love to hear what any others think!